client/src/components/Navbars/SidebarContents.tsx

```diff
@@ -134,21 +126,23 @@ function SidebarContents({ setShowSidebar }) {
       </Typography>
       <List>
         {topics && topics.length > 0 ? (
-          topics.map((topic, idx) => (
-            <RouterLink
-              to={`/visualiser/${titleToUrl(topic.title)}`}
-              style={{ color: 'inherit', textDecoration: 'none' }}
-            >
-              <ListItem key={idx} button>
-                <ListItemAvatar>
-                  <TopicIcon />
-                </ListItemAvatar>
-                <ListItemText
-                  primary={topic.title}
-                  secondary={<TagList tags={topic.courses} />}
-                />
-              </ListItem>
-            </RouterLink>
+          topics.map((topic, i) => (
```

Comment:
i was thinking of whether to use i or idx for the keys but using i as the key should be fine. just make sure the rest of the files are consistent

Reply:
Done.